plugins/lime-plugin-pirania/src/screens/voucher.spec.js

```diff
@@ -0,0 +1,78 @@
+import { h } from "preact";
+import { screen, act, fireEvent } from "@testing-library/preact";
+import { render } from "utils/test_utils";
+import "@testing-library/jest-dom";
+import { route } from "preact-router";
+import waitForExpect from "wait-for-expect";
+import Voucher from "./voucher";
+import { listVouchers } from "../piraniaApi";
+import queryCache from "utils/queryCache";
+import * as timeago from "timeago.js";
+
+jest.mock("../piraniaApi");
+const now = new Date().getTime() / 1000;
+let vouchers = [
+	{
+		code: "PIDFIG",
+		id: "x5crd4",
+		activation_date: now - 600e3,
+		expiration_date: now - 600e3 + 1200,
+		name: "luandro",
+		duration_m: 1200,
+		is_active: false,
+		creation_date: now - 700e3,
+		permanent: true
+	}
+];
+
+vouchers[0].status = vouchers[0].is_active
```

Comment:
The logic of how the voucher status is defined shouldn't be coded in the test file.

Reply:
Status coming from backend now.